test/parallel/test-url-revokeobjecturl.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+const common = require('../common');
```

Comment:
Declaring `common` is not needed because `common` is not used as @jasnell mentioned.
Refs: https://github.com/nodejs/node/blob/main/doc/contributing/writing-tests.md#lines-1-3
```suggestion
require('../common');
```